docs/Administration.rst

```diff
@@ -3061,6 +3064,136 @@ this entry is updated:
 -  The number of added users is increased if some users were added to the bound group by an administrator
 
 
+.. _AdminAppCenter:
+
+============================
+Managing Application Center
+============================
+
+As a platform administrator, you can manage the Application Center by:
+
+-  Defining the number of allowed favorite applications
+-  Defining mandatory applications that appear in every favorite applications drawer/list
+-  Defining default icon for added applications
+-  Adding/Editing applications
+
+To access the administration menu for Application Center, you need to access to the left navigation menu through
+the hamburger icon -> Administration -> OTHER -> Applications
+
+|AC_image0|
+
+In the administration screen for Application Center, you have two tabs: Applications and Settings
+
+|AC_image1|
+
+.. _SettingsTabAppCenter:
+
+Settings tab
+~~~~~~~~~~~~~
+
+|AC_image10|
+
+In the settings tab you can define the two properties:
+
+-  Number of favorites allowed: which defines the :ref:`maximum number of applications <max-fav-apps>` an end user is able to favorite
+
+-  Default app icon: which defines the icon that could take an application when it is added without an icon
+
+To define the above properties, you just need to click on the corresponding edit icon |AC_image2| in front of 
+the property you want to redefine.
+
+For the property **Number of favorites allowed**, you need either to move the cursor or click on +/- icons to reach
+the desired value then click on save icon |AC_image4|.
+
+|AC_image3|
+
+To set a custom default icon for application, other the one defined by eXo Platform, you need to follow these steps:
+
+-  click on the edit icon 
+-  browse for an image in your machine
+-  click on save icon to load the image in its location
+
+|AC_image5|
+
+If you need to edit the image again, you will find the previous image name instead of the browse button with a delete
+button allowing to restore the default icon.
+
+|AC_image6|
+
+.. _ApplicationsTabAppCenter:
+
+Applications tab
+~~~~~~~~~~~~~~~~~
+
+|AC_image11|
+
+Applications tab contains:
```

Comment:
The preview of this paragraph is KO.
The 2 dash are displayed on the same line, not on different lines

Reply:
Ok I'll check